docs/components/workflow.rst

```diff
@@ -1,28 +1,47 @@
 DKAN Workflow
 =============
 
-For large organizations, it can be difficult to moderate vast amounts of content submitted by a wide range of publishers and agencies. It is important to be able to manage which users are able to add, edit and delete content, as well as which users can view and approve of content under review.
+Introduction
+------
+For large organizations, it can be difficult to moderate vast amounts of content submitted by a wide range of publishers and agencies.
 
 DKAN Workflow is an optional module for `DKAN
 <https://github.com/GetDKAN/dkan>`_ based on the `Workbench
 <https://www.drupal.org/project/workbench>`_ family of modules.
 
+With Workflow, site managers and administrators can determine which users are able to add, edit and delete content, as well as which users can view and approve of content under review.
+
 Workflow creates a moderation queue so that content is published to the live site only after a designated supervisor or group moderator has reviewed and approved it. 
 
-*With Workflow, content exists in three states:*
+**When using DKAN Workflow, content exists in three states:**
 
-* _Draft_  - A saved work in progress.
-* _Needs_ _Review_ - The author feels the content is ready to go on public on the live site, and would like the supervisor to review it.
-* _Published_ - The content is public and visible on the live site.
+* **Draft**  - A saved work in progress.
+* **Needs Review** - The author feels the content is ready to go on public on the live site, and would like the supervisor to review it.
+* **Published** - The content is public and visible on the live site.
 
 .. figure:: ../images/workflow/dkan_workflow_screenshot.png
 
-   _What you see on My Workbench after login. The Workbench navigation bar contains your content, drafts, and more. The "Create Content" menu features a list of content types you can create.
+The above image displays you see on My Workbench after login. The Workbench navigation bar contains your content, drafts, and more. The "Create Content" menu features a list of content types you can create.
 
+Installation
+------------
 
-Requirements for DKAN Workflow
-------------------------------
+DKAN workflow is included on all out-of-the-box DKAN sites; however, it is not enabled by default. It can be enabled either from the Modules management page or by using drush.
 
+.. code-block:: bash
+
+   drush en dkan_workflow -y
+
+Enabling DKAN workflow will enable all of the necessary dependencies modules and update site
```

Comment:
> Enabling DKAN workflow will enable all of the necessary dependencies modules

Sounds weird, how about "Enabling DKAN workflow will automatically enable all other required modules and add the Workflow Supervisor role to all users already assigned the site manager role."